### ISSUE TITLE
[Bug]: Description Importing a text type file of 1 GB size succeeded. select...into outfile... failed.

### ISSUE THREAD

**Issue by tianyahui-python** [matrixorigin/matrixone#4733]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

create table t1 (a text);

source xxx/test2.sql

test2.sql size 1G.

select a from t1 into outfile 'xxx/csv' faileds terminated by ',';

error msg: ERROR 1105 (HY000): unsupported column type 252

**Comment by tianyahui-python** [comment]
mysql> create table t1(a text);
Query OK, 0 rows affected (0.02 sec)

mysql> 
mysql> insert into t1 values('sjdflisjgiojo239');
Query OK, 1 row affected (0.02 sec)

mysql> select * from t1 into outfile '/Users/tianyahui/PycharmProjects/local_test_project/sql/11.csv' fields terminated by ',';
ERROR 1105 (HY000): unsupported column type 252

**Comment by tianyahui-python** [comment]
@jianwan0214  bug continue exists.

**Comment by tianyahui-python** [comment]
mysql> create table t2(
    -> clo1 tinyint,
    -> clo2 smallint,
    -> clo3 int,
    -> clo4 bigint,
    -> clo5 tinyint unsigned,
    -> clo6 smallint unsigned,
    -> clo7 int unsigned,
    -> clo8 bigint unsigned,
    -> col9 float,
    -> col10 double,
    -> col11 varchar(255),
    -> col12 Date,
    -> col13 DateTime,
    -> col14 timestamp,
    -> col15 bool,
    -> col16 decimal(5,2),
    -> col17 text,
    -> col18 varchar(255),
    -> col19 varchar(255),
    -> col20 varchar(255),
    -> col21 varchar(255),
    -> col22 varchar(255),
    -> col23 varchar(255),
    -> col24 varchar(255),
    -> col25 varchar(255),
    -> col26 varchar(255),
    -> col27 varchar(255),
    -> col28 varchar(255),
    -> col29 varchar(255),
    -> col30 varchar(255),
    -> col31 varchar(255),
    -> col32 varchar(255),
    -> col33 varchar(255),
    -> col34 varchar(255),
    -> col35 varchar(255),
    -> col36 varchar(255),
    -> col37 varchar(255),
    -> col38 varchar(255),
    -> col39 varchar(255),
    -> col40 varchar(255),
    -> col41 varchar(255),
    -> col42 varchar(255),
    -> col43 varchar(255),
    -> col44 varchar(255),
    -> col45 varchar(255),
    -> col46 varchar(255),
    -> col47 varchar(255),
    -> col48 varchar(255),
    -> col49 varchar(255),
    -> col50 varchar(255),
    -> col51 varchar(255),
    -> col52 varchar(255),
    -> col53 varchar(255),
    -> col54 varchar(255),
    -> col55 varchar(255),
    -> col56 varchar(255),
    -> col57 varchar(255),
    -> col58 varchar(255),
    -> col59 varchar(255),
    -> col60 varchar(255),
    -> col61 varchar(255),
    -> col62 varchar(255),
    -> col63 varchar(255),
    -> col64 varchar(255),
    -> col65 varchar(255),
    -> col66 varchar(255),
    -> col67 varchar(255),
    -> col68 varchar(255),
    -> col69 varchar(255),
    -> col70 varchar(255),
    -> col71 varchar(255),
    -> col72 varchar(255),
    -> col73 varchar(255),
    -> col74 varchar(255),
    -> col75 varchar(255),
    -> col76 varchar(255),
    -> col77 varchar(255),
    -> col78 varchar(255),
    -> col79 varchar(255),
    -> col80 varchar(255),
    -> col81 varchar(255),
    -> col82 varchar(255),
    -> col83 varchar(255),
    -> col84 varchar(255),
    -> col85 varchar(255),
    -> col86 varchar(255),
    -> col87 varchar(255),
    -> col88 varchar(255),
    -> col89 varchar(255),
    -> col90 varchar(255),
    -> col91 varchar(255),
    -> col92 varchar(255),
    -> col93 varchar(255),
    -> col94 varchar(255),
    -> col95 varchar(255),
    -> col96 varchar(255),
    -> col97 varchar(255),
    -> col98 varchar(255),
    -> col99 varchar(255),
    -> col100 varchar(255)
    -> );
Query OK, 0 rows affected (0.09 sec)

mysql> load data infile '/Users/tianyahui/PycharmProjects/local_test_project/load_data/data/1000000_100_columns_load_data_1.csv' into table t2;
Query OK, 320665 rows affected (16.60 sec)

mysql> select count(*) from t2;
+----------+
| count(*) |
+----------+
|   320665 |
+----------+
1 row in set (0.11 sec)
mysql> select * from t2 into outfile '/Users/tianyahui/PycharmProjects/local_test_project/load_data/data/outfile_10000000.csv';
ERROR 20101 (HY000): internal error: unexpected mo-rpc address 127.0.0.1:6002
mysql> 
mysql>

**Comment by tianyahui-python** [comment]
@jianwan0214 bug continue exists.

**Comment by tianyahui-python** [comment]
92,29826,-217215902,-7764041449439486164,170,56970,2391306712,11993402294048455654,2.6087,5.1580,"I,G;KL/MN?OPQR.STU-_+=VWXYZabcd",9106-07-22,9651-03-04 00:00:00,2001-10-24 00:00:00,false,727.66,",GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123456","/MN?OPQR.STU-_+=VWX","CDEF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigk","G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz01234567","QR.STU-_+=VWXYZabcdef",_+=VWX,QR.STU-_+=VWXYZabcdefghigklm,F,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrs,-_+=VWXYZabcdefghigklmnopq,QR.STU-_+=VWXY,"R.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz01234","MN?OPQR.STU-_+=","G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstu",";KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqr","I,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuv",CDEF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz012,KL/MN?OPQR.STU-_+=VWX,KL/MN?OPQR.STU-_+=VWXYZabcde,=VWXYZabcdefghig,EF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxy,"TU-_+=VWXYZabcdefghi","QR.STU-_+=VWXYZabcdefghigklmnopqr","PQR.STU-_+=","DEF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123456","U-_+=VWXYZabcdefghigklm",_+=VWXYZabcdefghigklmnopq,I,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxy,L/MN?OPQR.STU-_+=VWXYZabcdefgh,N?OPQR.STU-_+=VWXYZabcdefghigkl,I,G;KL/MN?OPQR.STU-_+=VW,".STU-_+=VWXYZa",",GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqr","STU-_+=VWXYZabcdefghigklmnop","F,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstu","QR.STU-_+=VWXYZabcdefghig",GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklm,TU-_+=,.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz012,_+=VWXYZabcdefghigklmnopqrstuvw,=VWXY,"MN?OPQR.STU-_+=VWXYZabcdefg","+=VWXYZabcdef","I,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0","DEF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrst","GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123456",DEF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopq,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123,+=VWXYZabcdefghigklmno,.STU-_+=VWXYZabc,.STU-_+=VWXYZabcdefghigklmno,"PQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz01234","KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz01234","/MN?OPQR.STU-_+=VWXYZ","KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz01234","HI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz012345",+=VWXYZabcdefghigklmno,KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmn,?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123456,+=,I,G;KL/MN?OPQR.STU-_+=VWX,"QR.STU-_+=VWXYZa","QR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123","STU-_+=VWXYZabc","OPQR.STU-_+=VWXYZabcdefghigk","EF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcd",PQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz012345,N?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123,-_+=VWXYZabc,F,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigk,STU-_+=VWX,"_+=VWXYZabcdefghigklmnopqrstuvwx","I,G;KL/MN?OPQR.STU-_+=VWXYZabcdef","BCDEF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdef",",G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqr","N?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuv",BCDEF,GHI,G;KL/MN?OPQR.STU-_+=,EF,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz01234567,,GHI,G;KL/MN?OPQR.STU-_+=VWXYZabcdefghigkl,+=VWXYZabcdefghigklmnopqrstu,R.STU-_+=VWXYZabcdefghigklmnopqrstuvwx,"QR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz01",";KL/MN?OPQR.STU-_+=VWXYZabcdefghigklmnopqrstuvwxyz0123456","=VWXYZabcdefghig","/MN?OPQR.STU-_+=VW"

**Comment by zhangxu19830126** [comment]
@jianwan0214 The problem of bigger messages in rpc needs to be fixed, the default is already 100MB

**Comment by m-schen** [comment]
I will fix the bug after my finishing the 0.6 work.

**Comment by m-schen** [comment]
Why doesn't mo rpc split messages that exceed the size into multiple messages and assemble them at the receiving place？
